src_c/include/sse2neon.h

```diff
@@ -4569,7 +4569,7 @@ FORCE_INLINE __m128 _mm_cvtpi32_ps(__m128 a, __m64 b)
 
 // Convert packed signed 32-bit integers in a to packed single-precision
 // (32-bit) floating-point elements, store the results in the lower 2 elements
-// of dst, then covert the packed signed 32-bit integers in b to
+// of dst, then convert the packed signed 32-bit integers in b to
 // single-precision (32-bit) floating-point element, and store the results in
 // the upper 2 elements of dst.
 //
```

Comment:
This is a vendored file from https://github.com/DLTcollab/sse2neon

I don't think we should be manually editing this one too much

Reply:
The typo is already fixed on that repo.